src/features/personDetails/personCreditsSlice.js

```diff
@@ -0,0 +1,39 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const personCreditsSlice = createSlice({
+  name: "personCredits",
+  initialState: {
+    personCredits: null,
+    status: "initial",
+  },
+  reducers: {
+    axiosPersonCredits: () => ({
+      status: "loading",
+      personCredits: null,
+    }),
+    axiosPersonCreditsSuccess: (_, { payload: personCredits }) => ({
+      status: "success",
+      personCredits,
```

Comment:
people: people.results,
      totalPages: people.total_pages,
      totalResults: people.total_results,
      
      for the pagination (in future)